homeassistant/components/alexa/smart_home.py

```diff
@@ -417,6 +435,9 @@ def properties_supported(self):
     def properties_retrievable(self):
         return True
 
+    def properties_proactively_reported(self):
+        return True
```

Comment:
This should depend on if it is enabled by the user or not ?

Reply:
No, it simply declares that the _AlexaInterface supports proactive reporting. Whether or not the component reports something is not specified here.